extensions/stackdriver/stackdriver.cc

```diff
@@ -534,6 +622,7 @@ bool StackdriverRootContext::recordTCP(uint32_t id) {
       evaluateExpressions(record_info.extra_log_labels);
       record_info.expressions_evaluated = true;
     }
+    fillAuthzDryRunInfo(record_info.extra_log_labels);
```

Comment:
Can you add a test for TCP as well?

Reply:
added TCP test mostly based on the TestStackdriverTCPMetadataExchange.